aad/Module.md

```diff
@@ -1,91 +1,33 @@
 ## Overview
-Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language 
-easily. It provides capability to perform management operations on `User` and `Group` resources in an Azure AD tenent 
+Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language. 
+It provides capability to perform management operations on `User` and `Group` resources in an Azure AD tenent 
 (Organization).
 
 The connector is developed on top of Microsoft Graph is a REST web API that empowers you to access Microsoft Cloud 
```

Comment:
Let's rephrase this as: " The connector uses the Microsoft Graph REST web API that provides the capability to access Microsoft Cloud service resources. This version of the connector only supports the operations on users and groups in an Azure AD"

Reply:
Fixed